diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 1234567..abcdefg 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -32,7 +32,7 @@ find_package(Fortran REQUIRED)
 find_package(PkgConfig REQUIRED)
 
 # Load CMake macros located in the project cmake subdirectory.
-set (CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+set (CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/src/cmake")
 
 include(DeclareCasacppComponent)
 include(GNUInstallDirs)
